lib/Launcher.js

```diff
@@ -86,12 +89,16 @@ class Launcher {
       chromeProcess.stderr.pipe(process.stderr);
     }
 
-    const waitForChromeToClose = new Promise(fulfill => {
+    const waitForChromeToClose = new Promise((fulfill, reject) => {
       chromeProcess.once('close', () => {
         // Cleanup as processes exit.
-        if (temporaryUserDataDir)
-          removeSync(temporaryUserDataDir);
-        fulfill();
+        if (temporaryUserDataDir) {
+          removeFolderAsync(temporaryUserDataDir)
+              .then(() => fulfill())
+              .catch(err => console.error(err));
```

Comment:
why not just let this throw, like removeSync would have done.

Reply:
@graingert to avoid `unhandledPromiseRejection`.

Reply:
presumably people are going to `await Page.pdf();` and thus catch any errors

Reply:
How's `Page.pdf` related to this code?

Reply:
@aslushnikov it's the API that calls the download

Reply:
@aslushnikov ah you're right. Edited comment